<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e4ecfa;</string>
  </property>
  <widget class="QTextBrowser" name="tb_chat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 5 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">backgorund-color: #E5E9F0;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #275efe;
	border-radius: 10px;
	color: #ffffff;
}
QPushButton:hover {
	background-color: #274fff;
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_chat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>에스코어 드림 5 Medium</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;
border-radius: 8px;
color: #000000;
padding-right: 3px;
padding-left: 3px;
padding-bottom: 1px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Friend List</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>131</width>
     <height>541</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">backgorund-color: #D8DEE9;
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>129</width>
      <height>539</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D8DEE9;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_chatName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>대화상대 : all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
